remindme-common/src/main/java/it/andreafailli/remindme/common/models/BaseEntity.java

```diff
@@ -0,0 +1,46 @@
+package it.andreafailli.remindme.common.models;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.springframework.data.annotation.Id;
+
+public abstract class BaseEntity {
+
+	@Id
+	private String id;
+
+	public BaseEntity() {}
+	
+	public BaseEntity(String id) {
+		this.id = id;
+	}
+
+	public String getId() {
+		return id;
+	}
+	
+	public void setId(String  id) {
+		this.id = id;
+	}	
+
+	@Override
+	public boolean equals(Object obj) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This class overrides "equals()" and should therefore also override "hashCode()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1206)